{{ cookiecutter.project_slug }}/.markdownlint.yaml

```diff
@@ -1,2 +1,3 @@
 # Disable checks here
-MD013: false
+MD013:
+  line_length: 90
```

Comment:
I'd normally go with 100 for MD files, but it's not worth bikeshedding about :stuck_out_tongue:.

Does markdownlint wrap long lines btw? I think the reason we usually turn this option off is so that we don't have to manually rewrap lines.

I use this extension to autowrap lines: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

Maybe we want to include it in `extensions.json`? (I think it relies on where you've set your editor rulers to be in `settings.json` though, so maybe we want to change the rulers for MD files while we're at it.)